crucible-llvm/src/Lang/Crucible/LLVM/Translation/Instruction.hs

```diff
@@ -447,21 +449,21 @@ evalGEP instr (GEPResult _lanes finalMemType gep0) = finish =<< go gep0
          traverse (\(x,i) -> calcGEP_array mt' x i) (Seq.zip xs idxs)
 
 
-calcGEP_array :: forall wptr arch s ret.
+calcGEP_array :: forall wptr s arch ret.
```

Comment:
Is this to fix some type application ordering?

Reply:
This change is not required and superfluous: I had initially added an `ext` here and subsequently removed it, but in the process of determining where to add it I had re-ordered this to match the order of introduction.  I can revert this change if desired.